app/views/devise/registrations/new.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= f.label :name %><br />
     <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
   </div>
-
+  
   <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
```

Comment:
Regarding the db/.DS_Store file below, this should be removed from this branch and added to .gitignore. After you answer the other questions/concerns in this review, ping me and I'll help you fix this!

Reply:
deleted the file and added to git.ignore